functional/FunctionalTest.java

```diff
@@ -2473,12 +2478,23 @@ public static void main(String[] args) {
       client = new MinioClient(endpoint, accessKey, secretKey);
       // Enable trace for debugging.
       // client.traceOn(System.out);
-      FunctionalTest.runTests();
-
-      // Run fast test with region parameter passed to the constructor
-      client = new MinioClient(endpoint, accessKey, secretKey, region);
-      FunctionalTest.runFastTests();
 
+      // For mint environment, run tests based on mint mode
+      if (mintEnv) {
+        if (mintMode != null && mintMode.equals("full")) {
+          FunctionalTest.runTests();
+        } else {
+          FunctionalTest.runQuickTests();
+        }
+      } else {
+        FunctionalTest.runTests();
+
+        // Get new bucket name to avoid minio azure gateway failure.
+        bucketName = getRandomName();
+        // Quick tests with passed region.
+        client = new MinioClient(endpoint, accessKey, secretKey, region);
```

Comment:
Also this line, what is this for?

Reply:
> I am unable to understand your question

I mean isn't this line of code useless ? client seems like an unused variable for me.

Reply:
Nope. You missed `client` is `private static` to `public class FunctionalTest` and all tests use `client`

Reply:
> Nope. You missed client is private static to public class FunctionalTest and all tests use client

okay.. also, it is little weird that the client is being instantiated many times in main(). I guess we can see that later.